PE-7: Clean up video-mode-plugin repo to make it more clear and understandable

mxin512 opened the pull request:
- added a few details on why and what is video mode plugin feature
- cleaned up videomode_plugin.brs and Plugin Message Usage

mxin512 committed:
PE-7: Clean up video-mode-plugin repo to make it more clear and understandable

- added a few details on why and what is video mode plugin feature
- cleaned up videomode_plugin.brs and Plugin Message Usage

bhastings1019 commented on README.md:
```diff
@@ -1,111 +1,23 @@
-video-mode-plugin
-=================
+# video-mode-plugin
+The Video Mode Plugin feature is designed for allowing customers to set the BrightSign videoMode.
```
Suggest adding something along these lines to explain what a video mode plugin does for you and why someone may want or need it, `to a resolution that is non-standard or beyond the current list of video modes in BrightAuthor:connected or BrightAuthor. `.

mxin512 replied:
Updated in [8ca7cbd](https://github.com/brightsign/video-mode-plugin/pull/2/commits/8ca7cbd5dbbb9576d85d01fc2d59c0943daa807e)

mxin512 committed:
PE-7: Clean up video-mode-plugin repo to make it more clear and understandable

- added a few details on why and what is video mode plugin feature
- cleaned up videomode_plugin.brs and Plugin Message Usage
